django_lightweight_queue/utils.py

```diff
@@ -8,6 +10,31 @@
 
 from . import app_settings
 
+SETTING_NAME_PREFIX = 'LIGHTWEIGHT_QUEUE_'
```

Comment:
We should probably only define `SETTING_NAME_PREFIX` in one place (it's in app_settings atm too).